project-1-flexbox/Style.css

```diff
@@ -0,0 +1,188 @@
+
+
+
+* {
+    box-sizing: border-box;
+}
+
+
+
+
+
+
+
+
+.container {
+	padding: 30px;
+	margin-right:20px
+
+	
+}
+
+
+
+.header {
+	display: flex;
+	flex-direction: row; 
+	
+
+}
+nav {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+}
+ul {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	width:1000px;
+	list-style: none;
+}
+header img {
+	padding-left: 120px;
+    
+}
+a {
+	text-decoration: none;
+	color: black;
+}
+
+.section1 {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-end;
+	margin-top: 100px;
+	margin-right:2px;
+
+
+
+   
+}
+
+.img1 {
+	padding-left: 300px;
+
+
+}
+
+h1 {
+	font-size: 70px;
+
+	text-align: center; 
+	
+}
+
+
+.div1{
```

Comment:
You should try to keep similarities across the document. if you have one section that's
```
.class {
     attribute: value;
     attribute2: value2;
}
```
you should try and keep it that way.